libs/designsystem/src/lib/components/chart/configs/type.config.ts

```diff
@@ -98,4 +109,114 @@ export const CHART_TYPES_CONFIG: ChartTypesConfig = {
       },
     },
   },
+  stock: {
+    type: 'line',
+    options: {
+      // locale is overridden in handleLocalization().
+      locale: 'en-US',
+      responsive: true,
+      animation: {
+        duration: 0,
+      },
+      layout: {
+        padding: {
+          left: 0,
+          right: 10,
+          top: 30,
+          bottom: 0,
+        },
+      },
+      backgroundColor: getThemeColorRgbString('semi-light', 0.5),
+      scales: {
+        x: {
+          type: 'time',
+          // Time is overridden in handleLocalization().
+          time: {},
+          grid: {
+            lineWidth: 0,
+          },
+          ticks: {
+            maxRotation: 0,
+            source: 'data',
+            autoSkipPadding: 40,
+            font: {
+              size: parseInt(fontSize('xs')),
+            },
+          },
+        },
+        y: {
+          position: 'right',
+          display: true,
+          grid: {
+            drawBorder: false,
+          },
+          ticks: {
+            display: true,
+            font: {
+              size: parseInt(fontSize('xs')),
+            },
+          },
+        },
+      },
+      elements: {
+        line: {
+          tension: 0, // Smooth curve (0 = no smoothing)
+          borderWidth: 2,
+        },
+        point: {
+          hitRadius: 20,
+          radius: 0,
+          hoverRadius: 0,
+          hoverBorderWidth: 0,
+        },
+      },
+      normalized: true,
+      plugins: {
+        tooltip: {
+          mode: 'index',
+          intersect: false,
+          caretSize: 1,
+          callbacks: {
+            labelColor: (tooltipItem: TooltipItem<keyof ChartTypeRegistry>) => {
+              return {
+                backgroundColor: tooltipItem.dataset.borderColor,
+                borderColor: tooltipItem.dataset.borderColor,
+                borderWidth: 0,
+              } as TooltipLabelStyle;
+            },
+          },
+        },
+        crosshair: {
+          line: {
+            color: 'red',
+          },
+          sync: {
+            enabled: false,
+          },
+        },
+
+        datalabels: {
+          backgroundColor: (context: Context) => context.dataset.borderColor as Color,
+          color: getThemeColorHexString('white'),
+          borderRadius: 3,
+          font: {
+            lineHeight: 1,
+            size: parseInt(fontSize('xs')),
+          },
+          padding: {
+            top: 6,
+            left: 5,
+            right: 5,
+            bottom: 5,
+          },
+          offset: 5,
+          align: (context: Context): Align =>
+            (getChartPointFromContext(context) as any)?.datalabel.position,
+          display: (context: Context): boolean =>
+            !!(getChartPointFromContext(context) as any)?.datalabel,
+          formatter: (value: any, context: Context): string => value.datalabel.value,
```

Comment:
My linter gives me: `[tsserver 6133] [I] 'context' is declared but its value is never read.`
```suggestion
          formatter: (value: any): string => value.datalabel.value,
```